app/assets/stylesheets/automated_tests.scss

```diff
@@ -1,20 +1 @@
-@import 'common/core';
-
-
-fieldset {
-  box-shadow: none;
-
-  // Save button in assignment form
-  & + input[type='submit'] {
-    margin-top: 1.5em;
-  }
-
-  fieldset legend span {
-    padding-top: 0;
-    top: 0.5em;
-  }
-}
-
-#test_properties {
-  margin-top: 2em;
-}
+@import 'assignments'
```

Comment:
Files should end with a trailing newline<br>Prefer double-quoted strings<br>Declaration should be terminated by a semicolon